### PR TITLE
Ubuntu 20.04 Dockerfile for Github Actions

### PR DESCRIPTION
**Pull request by asenyaev**

### CODE REVIEW

**Comment by asenyaev** [comment]
@asmorkalov, @fengyuentau, please, take a look!

@fengyuentau, could you add an ARM64 Dockerfile?

**Comment by fengyuentau** [comment]
> could you add an ARM64 Dockerfile?

Yes, it is here https://github.com/opencv-infrastructure/opencv-worker-config/pull/2.

File: images/ubuntu-github-actions--20.04/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    libsuitesparse-dev \
+    liblapacke-dev \
+    libceres-dev \
+    libeigen3-dev \
+    libgtkglext1-dev \
+    libopenni2-dev \
+    libglfw3-dev \
+    xvfb \
+    pkg-config && \
+  apt-get clean && python3 -m pip install --upgrade pip && pip install requests pylint==2.4.4
```
Comment (fengyuentau) [inline images/ubuntu-github-actions--20.04/Dockerfile]: Clean pip cache as well?

Comment (asenyaev) [reply images/ubuntu-github-actions--20.04/Dockerfile]: Sure. Added pip cache purge.

**Comment by asenyaev** [comment]
Added also `rsync` for extended DNN testing changes.

**Comment by asenyaev** [comment]
Closing PR. The dockerfile is in [opencv-infrastructure/opencv-gha-dockerfile](https://github.com/opencv-infrastructure/opencv-gha-dockerfile).

**Pull request closed without merging**